### PR TITLE
Outpath

### LINKED ISSUE #8: Allow to set out folder in fezc

### PR DESCRIPTION
**Pull request by forki**
fixes #8 on windows

### CODE REVIEW

**Comment by forki** [comment]
also better error logging (copied from paket)

File: out/interop.core [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module 'interop' [
```
Comment (kjnilsson) [inline out/interop.core]: Was this meant to be included?

File: test/test.fsproj [hunk 2]
```diff
@@ -9,6 +9,7 @@
     <Compile Include="aether_test.fs" />
     <Compile Include="wip.fs" />
     <Compile Include="basics.fs" />
+    <Compile Include="interop.fs" />
```
Comment (kjnilsson) [inline test/test.fsproj]: This code is already in basics.fs do we need another file?

Comment (forki) [inline test/test.fsproj]: want to add more of this at later point

Comment (forki) [inline test/test.fsproj]: removed for now

**Comment by forki** [comment]
added a bit of docs ;-)

File: src/fez/Program.fs [hunk 3]
```diff
@@ -37,11 +83,13 @@ let main argv =
                       |> writeCoreFile dir n
                       |> outFiles.Add
 
-        for file in outFiles do
-            printfn "%s" file
+        if not noBeam then
+            for outDir,file in outFiles do
```
Comment (kjnilsson) [inline src/fez/Program.fs]: Why not pass all files to erlc in one go?

Comment (forki) [inline src/fez/Program.fs]: Yes good idea

Comment (forki) [inline src/fez/Program.fs]: done

**Comment by kjnilsson** [comment]
Awesome thanks.

**Pull request merged**